src/GcmChannel.php

```diff
@@ -100,7 +101,7 @@ protected function handleFailedNotifications($notifiable, Notification $notifica
             }
 
             $this->events->fire(
-                new NotificationFailed($notifiable, $notification, $this, [
+                new NotificationFailed($notifiable, $notification, get_class($this), [
```

Comment:
This is for this issue reported:

https://github.com/laravel-notification-channels/gcm/issues/15